build.js

```diff
@@ -1,17 +1,52 @@
-var fs = require("fs");
-var path = require("./paths");
-var fse = require("fs-extra");
-
-path.forEach((e) => {
-  var sourceDir = e.source;
-  var destDir = `public/${e.destination}`;
-
-  if (!fs.existsSync(destDir)) {
-    fs.mkdirSync(destDir, { recursive: true });
-  }
-
-  fse.copy(sourceDir, destDir, function (err) {
-    if (err) console.error(err);
-    else console.log("success!", e.destination);
-  });
+const fs = require('fs');
+const fse = require('fs-extra');
+const tools = require('./tools');
+
+const storybookSetup = `
+  export default {
+    title: 'Bluegenes',
+  };
+  
+  import React from 'react';
+  import ReactDOM from 'react-dom';
+  
+  window.React = React;
+  window.ReactDOM = ReactDOM;
+
+  import imjs from 'imjs';
+  window.imjs = imjs;
+  import { renderTool } from '../util';
+  `;
+  
+fs.mkdirSync('stories', { recursive: true });
+fs.mkdirSync('configs', { recursive: true });
+
+fs.writeFileSync(`stories/index.stories.js`, storybookSetup);
+
+tools.map((tool) => {
+	const config = JSON.parse(fs.readFileSync(`node_modules/${tool.name}/config.json`));
+	const toolName = config.toolName.cljs;
+	fs.writeFileSync(`configs/${toolName}.json`, JSON.stringify(config));
+
+	const distSource = `node_modules/${tool.name}/dist`;
+	const distDestination = `public/${toolName}`;
+	if (!fs.existsSync(distDestination)) {
+		fs.mkdirSync(distDestination, { recursive: true });
+	}
+
+	fse.copy(distSource, distDestination, function (err) {
+		if (err) console.error(err);
+		else console.log('success!', distDestination);
+	});
+
+	const title = toolName.substr(9, toolName.length);
```

Comment:
> personally I wish none of the tools had this prefix

Just want to add that I mean this comment for `toolName.human`. `toolName.cljs` **should** be prefixed with bluegenes or something else that isn't likely to be taken, as it defines a global var. This means you can ignore this comment: `Also if https://github.com/intermine/generator-bluegenes-tool/ encourages you to name your tool prefixed with "bluegenes", please let me know!`


I still think it would be good to handle this, as if it had a name shorter than 9 characters it would be an empty string, which would probably cause things to fail.